### ISSUE TITLE
Zeroing the F/T Sensor?

### ISSUE THREAD

**Issue by danzimmerman** [UniversalRobots/Universal_Robots_ROS2_Driver#447]
Is there a built-in way to zero the F/T sensor from the driver to get rid of no-load offsets before carrying out operations that use the sensor? I get a few Newtons of force with no tool or payload, within the +/- 4N spec but I'd like to zero it out from time to time.

I know there's a `zero_ft_sensor()` URScript command but
* Looking at the features docs [here](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/blob/main/ur_robot_driver/doc/features.rst) it appears we can't send isolated script commands.
* Looking at `/ur_robot_driver/resources/ros_control.urscript` (referenced in launch [here](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/blob/main/ur_robot_driver/launch/ur_control.launch.py#L84)) it seems like the running program is just handling the speed or position commands.
* It doesn't seem like `/dashboard_client` or `/io_and_status_controller` provide anything to do this.

In the absence of a way to call `zero_ft_sensor()` I can subscribe to `/force_torque_sensor_broadcaster/wrench` for a while and subtract an average offset wrench from the measured wrench. Maybe I could also zero the in my Polyscope robot program prior to the External Control node running?

However, I wanted to check that I'm not missing something in the available interfaces.

**Comment by fmauch** [comment]
This is correct. I'll add this issue to the ROS1 feature parity Milestone. This is directly depending on #251.
